packages/engine-core/src/data-proxy/utils/request.ts

```diff
@@ -40,7 +40,6 @@ function buildHeaders(options: RequestOptions): RequestOptions['headers'] {
   return {
     ...options.headers,
     'Content-Type': 'application/json',
-    'Content-Length': `${options.body?.length ?? 0}`,
```

Comment:
This not being necessary and wrong, we remove it as part of the fix for https://prisma-company.slack.com/archives/C9D3H0P1A/p1636133659072700